libs/domain/user/address-list/src/address-list.component.ts

```diff
@@ -46,9 +49,17 @@ export class AddressListComponent extends ComponentMixin<AddressListItemOptions>
     distinctUntilChanged()
   );
 
-  protected addresses$ = combineLatest([
-    resolve(AddressService).getAddresses().pipe(distinctUntilChanged()),
-    new ContentController(this).getOptions(),
+  protected addresses$ = this.addressService
+    .getAddresses()
+    .pipe(distinctUntilChanged());
```

Comment:
`distinctUntilChanged` doesn't make sense for object

Reply:
Yeah, my bad. I refactored it but didn't pay attention on observables logic